deploy/manager/10_scylla_cluster.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  name: scylla-manager-cluster
+  namespace: scylla-manager
+spec:
+  agentVersion: 2.2.0
+  datacenter:
+    name: manager-dc
+    racks:
+    - members: 1
+      name: manager-rack
+      resources:
+        limits:
+          cpu: 1
+          memory: 200Mi
+        requests:
+          cpu: 1
+          memory: 200Mi
+      scyllaAgentConfig: scylla-agent-config
+      scyllaConfig: scylla-config
+      storage:
+        capacity: 5Gi
+  developerMode: true
```

Comment:
if this is the prod manifest, we shouldn't use developer mode

Reply:
It's consistent with https://github.com/scylladb/scylla-manager/blob/master/dist/scripts/scyllamgr_setup#L113, full performance of his cluster isn't needed.